sources/packages/web/src/services/http/ConfirmationOfEnrollmentApi.ts

```diff
@@ -62,4 +67,34 @@ export class ConfirmationOfEnrollmentApi extends HttpBaseClient {
       throw error;
     }
   }
+
+  public async getCOEDenialReasons(): Promise<COEDeniedReasonDto> {
+    try {
+      const response = await this.apiClient.get(
+        `institution/location/confirmation-of-enrollment/denial-reasons`,
```

Comment:
Just as recommendation, since there is no string interpolation, the code could be changed as below:
```
"institution/location/confirmation-of-enrollment/denial-reasons"
```